experiments/train_se3_diffusion.py

```diff
@@ -158,6 +178,13 @@ def model(self):
     def conf(self):
         return self._conf
 
+    def get_ddp_info(self):
```

Comment:
This shouldn't be a instance method since it never accesses `self`. Remove it from the class.

i.e. put it at the top of the file

def _get_ddp_info():
     .......